### PR TITLE
Add JDME for the years -1000 to +1000

### PR DESCRIPTION
**Pull request by westy92**
Taken from the same reference book.

### CODE REVIEW

File: src/delta_time.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ pub fn delta_time(year: f64) -> f64 {
     };
     let u = k + (year - 2000.0) / 100.0;
 
-    a0 + a1 * u.powi(1)
+    a0 + a1 * u
```
Comment (westy92) [inline src/delta_time.rs:42]: `.powi(1)` was essentially a no-op, right?

**Comment by vinc** [comment]
Hi @westy92, thanks a lot for your contribution :+1: 

I've been using `geodate` for my personal calendar (https://github.com/vinc/geocal) for a few years so I need to check the changes for various dates to see if and how they impact conversions to the Gregorian calendar. I'll get back to you after that :)

File: src/delta_time.rs (hunk 1 above)
Comment (vinc) [reply src/delta_time.rs:42]: Indeed we can remove it, it was only there for the symmetry with the next operations but it's really not needed.

**Comment by vinc** [comment]
Would you mind pulling the latest changes from the main branch to have the tests running on the PR? Thanks!

**Comment by westy92** [comment]
All set - it just needs your approval to run :)

**Comment by westy92** [comment]
CI passed 🎉

**Comment by westy92** [comment]
@vinc any update on this?

**Comment by westy92** [comment]
@vinc any update on this?

**Comment by westy92** [comment]
@vinc any update on this?

**Comment by westy92** [comment]
@vinc any update on this?

**Comment by vinc** [comment]
Sorry for the long delay.. Can you explain how you will use the change?

Your PR improves some calculations for the years -1000 to 1000 but this tool can't go before 1623 due to the way we compute delta time. See here:

https://github.com/vinc/geodate/blob/master/src/geodate.rs#L28
https://github.com/vinc/geodate/blob/master/src/delta_time.rs#L15

From the (very interesting) paper of the algo I'm using it seems hard to have any kind of real accuracy before that time and the invention of the telescope:

https://www.researchgate.net/publication/275069074_Delta_T_Polynomial_Approximation_of_Time_Period_1620-2013

**Comment by westy92** [comment]
Hi Vincent,

I use these directly:
use geodate::{earth_orbit, moon_phase, sun_transit};

I don't actually use the geodate format, just some of its internals.

Cheers,
Seth

**Review by vinc — APPROVED** [review]

**Pull request merged**